.github/workflows/build.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - main
+      - typefox-dev
   pull_request:
     branches:
       - main
+      - typefox-dev
```

Comment:
PR's are not really needed, we can push directly into `typefox-dev`. WDYT?

Reply:
Ok